DOC: Add What's New for PR 17842 (Algorithm Improvements for Fast Lomb Scargle) (#19655)
astropy/astropy: docs/whatsnew/8.0.rst

str1 float64 str1
---- ------- ----
   x     2.0    a

.. _whatsnew-8.0-timeseries-lra:

Fast Lomb-Scargle: Low Rank Approximation
=========================================

Fast Lomb-Scargle Periodogram (``fast`` and ``fastchi2`` implementations)
now uses Low Rank Approximation (LRA) algorithm by default.
The LRA implementation is up 4x slower, when compared to the existing one,
but also 8 to 10 orders of magnitude more accurate, nearly matching the
accuracy of their slow (non `fast`) counterparts.

This change of default allows eliminating previously seen issues with negative powers.

Additionally, the Bessel function approximation now relies on a Taylor

> expansion around ``x=0`` and as such no longer relies on SciPy for evaluation.

Reviewer: as per @kkrutkowski's remark, this isn't a change relative to any previously released version, so it shouldn't be listed here
```suggestion
```

str1 float64 str1
---- ------- ----
   x     2.0    a

.. _whatsnew-8.0-timeseries-lra:

Fast Lomb-Scargle: Low Rank Approximation
=========================================

Fast Lomb-Scargle Periodogram (``fast`` and ``fastchi2`` implementations)
now uses Low Rank Approximation (LRA) algorithm by default.
The LRA implementation is up 4x slower, when compared to the existing one,
but also 8 to 10 orders of magnitude more accurate, nearly matching the

> accuracy of the slow method.

Reviewer: The sentence as a whole doesn't make sense to me. Maybe that's what is actually meant here ?
```suggestion
accuracy of their slow (non `fast`) counterparts.
```

While not having a significant impact on the typical accuracy of the
standard fast implementation (except eliminating the negative powers issue),
it significantly improves the ``fastchi2`` method, causing it to nearly

> match the slow ``chi2`` implementation in the terms of accuracy.

Reviewer: Rephrased an shorten, since the second part is almost completely redundant with the previous paragraph (if I understand correctly)
```suggestion
This change of default allows eliminating previously seen issues with negative powers.
```

Additionally, the Bessel function approximation relies on the Taylor

> series around ``x=0`` and as such no longer relies on SciPy for evaluation.

Reviewer: ```suggestion
Additionally, the Bessel function approximation now relies on a Taylor
expansion around ``x=0`` and as such no longer relies on SciPy for evaluation.
```
